src/AWS.Deploy.Orchestration/Data/AWSResourceQueryer.cs

```diff
@@ -456,10 +456,71 @@ public async Task<List<Vpc>> GetListOfVpcs()
                 .ToListAsync());
         }
 
-        public async Task<List<PlatformSummary>> GetElasticBeanstalkPlatformArns()
+        public static void SortElasticBeanstalkWindowsPlatforms(List<PlatformSummary> windowsPlatforms)
```

Comment:
could you please add some documentation for this method to explain how the sorting is performed?

Reply:
Done